src/yarpWholeBodyInterface.cpp

```diff
@@ -172,3 +195,144 @@ yarp::os::Mutex& yarpWholeBodyInterface::getInterfaceMutex()
 {
     return wbiMutex;
 }
+
+bool yarpWholeBodyInterface::removeActuator(const wbi::ID& j)
+{
+    return actuatorInt->removeActuator(j);
+}
+
+bool yarpWholeBodyInterface::addActuator(const wbi::ID& j)
+{
+    return actuatorInt->addActuator(j);
+}
+
+int yarpWholeBodyInterface::addActuators(const wbi::IDList& j)
+{
+    return actuatorInt->addActuators(j);
+}
+
+const wbi::IDList& yarpWholeBodyInterface::getActuatorList()
+{
+    return actuatorInt->getActuatorList();
+}
+bool yarpWholeBodyInterface::setControlMode(wbi::ControlMode cm, double* ref,
+    int jnt)
+{
+    return actuatorInt->setControlMode(cm, ref, jnt);
+}
+bool yarpWholeBodyInterface::setControlReference(double* ref, int jnt)
+{
+    return actuatorInt->setControlReference(ref, jnt);
+}
+bool yarpWholeBodyInterface::setControlParam(wbi::ControlParam parId, const void* val, int jnt)
+{
+    return actuatorInt->setControlParam(parId, val, jnt);
+}
+
+// STATES
+bool yarpWholeBodyInterface::addEstimate(const wbi::EstimateType st,
+    const wbi::ID& sid)
+{
+    return false;
+}
+int yarpWholeBodyInterface::addEstimates(const wbi::EstimateType st,
+    const wbi::IDList& sids)
+{
+    return 0;
+}
+bool yarpWholeBodyInterface::removeEstimate(const wbi::EstimateType st,
+    const wbi::ID& sid)
+{
+    return false;
+}
+const wbi::IDList&
+yarpWholeBodyInterface::getEstimateList(const wbi::EstimateType st)
+{
+    return empty_id_list;
+}
+int yarpWholeBodyInterface::getEstimateNumber(const wbi::EstimateType st)
+{
+    return 0;
+}
+bool yarpWholeBodyInterface::getEstimate(const wbi::EstimateType et,
+    const int estimate_numeric_id,
+    double* data, double time,
+    bool blocking)
+{
+    return stateInt->getEstimate(et, estimate_numeric_id, data, time, blocking);
+}
+bool yarpWholeBodyInterface::getEstimates(const wbi::EstimateType et,
+    double* data, double time,
+    bool blocking)
```

Comment:
Not that we have to change this, but just out of curiosity: Is this part of the WebKit guidelines? 
When I break argument list I prefer to do it indenting on the first argument:
~~~
bool yarpWholeBodyInterface::getEstimates(const wbi::EstimateType et,
                                                                         double* data, double time, 
                                                                         bool blocking)
~~~
As I found it more readable (you read the argument list without moving the eye).